app/src/main/java/com/android/unio/model/event/EventRepositoryFirestore.kt

```diff
@@ -50,6 +51,16 @@ class EventRepositoryFirestore @Inject constructor(private val db: FirebaseFires
     }
   }
 
+  /**
+   * Fetches a [DocumentReference] for an [Event] object using the provided [uid].
+   *
+   * @param uid [String] : The uid of the [Event] to fetch.
+   * @return [DocumentReference] : The [DocumentReference] for the [Event] object.
+   */
+  override fun getEventRef(uid: String): DocumentReference {
+    return db.collection(EVENT_PATH).document(uid)
+  }
```

Comment:
I did an abstract version of this function in a utils file in my new PR #298. If I merge it before and you have time, you can just replace it in the blink of an eye. Otherwise it'll be a small refactor.

Reply:
I will do a quick refactor next sprint, as getAssociationRef and getUserRef could also be replaced with the abstract version.